Updates for several July 2023 mc releases (#964)
minio/docs: source/reference/minio-mc/mc-support-top-net.rst

=======================
``mc support top net``
=======================

.. default-domain:: minio

.. contents:: Table of Contents
   :local:
   :depth: 2

.. mc:: mc support top net

Syntax
------

> .. start-mc-support-top-net-desc

Reviewer: Stretch scope here - we have an admonition on `mc support` about these requiring SUBNET, but we don't have that on subpages.

We should maybe have it on all the subpages right after the description.

Author: I added it, though I need to double check because i think at least `proxy` does NOT require the registration.

The :mc:`mc support top net` command displays realtime network metrics.

.. end-mc-support-top-net-desc

.. tab-set::

   .. tab-item:: EXAMPLE

      The following command displays the current realtime network metrics for the :term:`alias` ``myminio`` deployment.

      .. code-block:: shell
         :class: copyable

         mc support top net myminio/

      The output returns information such as the server URL, interface, receive, transmit, and messages.

   .. tab-item:: SYNTAX

      The command has the following syntax:

      .. code-block:: shell
         :class: copyable

         mc [GLOBALFLAGS] support top disk                \
                                      [--interval value]  \
                                      TARGET

      .. include:: /includes/common-minio-mc.rst
         :start-after: start-minio-syntax
         :end-before: end-minio-syntax

Parameters
~~~~~~~~~~

.. mc-cmd:: TARGET
   :required:

   The full path to the :ref:`alias <minio-mc-alias>` or :term:`prefix` where the command should run.

.. mc-cmd:: --interval
   :optional:

   The interval in seconds between metric requests.

>    If no entry is made, the command requests metrics every second.

Reviewer 2: Idea that removes a few words:
```suggestion
   By default, the command requests metrics every second.
```

=======================
``mc support top net``
=======================

.. default-domain:: minio

.. contents:: Table of Contents
   :local:
   :depth: 2

.. mc:: mc support top net

.. include:: /includes/common-mc-support.rst
   :start-after: start-minio-only
   :end-before: end-minio-only

Syntax
------

.. start-mc-support-top-net-desc

The :mc:`mc support top net` command displays realtime network metrics.

.. end-mc-support-top-net-desc

.. tab-set::

   .. tab-item:: EXAMPLE

      The following command displays the current realtime network metrics for the :term:`alias` ``myminio`` deployment.

      .. code-block:: shell
         :class: copyable

         mc support top net myminio/

>       The output returns information such as the server URL, interface, receive, transmit, and messages.

Reviewer: Nit - this reads a little weird?

```suggestion
      The output returns information such as the server URL, network interface, receive rate, transmit rate, and messages.
```

Which...what are messages? but eh maybe its fine.
